fdbserver/BlobWorker.actor.cpp

```diff
@@ -1107,7 +1107,7 @@ ACTOR Future<BlobFileIndex> dumpInitialSnapshotFromFDB(Reference<BlobWorkerData>
 			}
 			retries++;
 			CODE_PROBE(true, "Granule initial snapshot failed");
-			// FIXME: why can't we supress error event?
+			// FIXME: why can't we suppress error event?
```

Comment:
If this is asking why something like `.error(e).suppressFor(...)` doesn't work, it's because both of these functions can suppress the event, and I think there can be some weird interactions from calling them in sequence (I don't remember the details, though). At any rate, you can get around this by calling `.errorUnsuppressed(e).suppressFor(...)`, which works the same way except that it doesn't filter the actor cancelled error. If you want that error to be filtered, you would need to do so with your own if statement.